docs/user-guide/dream/dream-data-reduction.ipynb

```diff
@@ -148,11 +148,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "to_save = dspacing_histogram.assign_coords(\n",
-    "    dspacing=sc.midpoints(\n",
-    "        dspacing_histogram.coords[\"dspacing\"]\n",
-    "    ))\n",
-    "scn.io.save_xye(\"dspacing.xye\", to_save)"
+    "cif_data.comment = \"\"\"This file was generated with the DREAM data reduction user guide\n",
```

Comment:
Additional metadata should be added (if not stored somewhere else in the CIF file):
- version of essdiffraction used
- date of creation
- instrument name and detector bank
- sample name
- raw files: sample and vanadium (if used)
- values and units of average 2theta, average Ltotal, dtt1 or difc

Reply:
I agree. But that should be a separate PR. In particular because we don't even know what some metadata (esp. sample) looks like in the input.

But note that this is filled in automatically:
```
_audit.creation_date 2024-11-14T16:20:10+00:00
_audit.creation_method 'Written by scippneutron v24.11.0'
_computing.diffrn_reduction 'ess.dream v24.10.1.dev32+gacad30d.d20241114'

_diffrn_radiation.probe neutron
_diffrn_source.beamline DREAM
_diffrn_source.facility ESS
_diffrn_source.device spallation
```